Makefile

```diff
@@ -240,7 +242,7 @@ else
 SCRUB_COMMAND =
 endif
 
-SENTRY_DSN_UI ?=
+SENTRY_DSN_UI ?= https://ff836b1e4abc4d0699bcaaf07ce4ea08@sentry.io/1366139
```

Comment:
will this send my local issues to the prod sentry instance? Isn't this token secret?

Reply:
@Hades32 We already had this in `apps/livechat/Makefile`, the idea was to move it here for simplicities sake. I wasn't sure if it was supposed to be a secret, but if so we probably need to regenerate it, it's been in clear text in this repo for ages 😓 

I get your point about it affecting local devenvs, we should explicitly set this to nothing for devenvs.

Reply:
Oh, actually, we're not `export`ing it so I don't believe it's being passed to `dev-ui` or `dev-livechat`. It is being explicitly passed to `build-ui` and `build-livechat` which is to be expected.